lib/Models/BoxDrawing.ts

```diff
@@ -619,6 +626,19 @@ export default class BoxDrawing {
     const eventHandler = new ScreenSpaceEventHandler(this.scene.canvas);
     eventHandler.setInputAction(handlePick, ScreenSpaceEventType.LEFT_DOWN);
     eventHandler.setInputAction(handleRelease, ScreenSpaceEventType.LEFT_UP);
+    Object.values(KeyboardEventModifier).forEach(
+      // Bind the release action to all LEFT_UP + any modifier. This is
+      // required because we want the release to happen even if the user is by
+      // chance pressing down on some other key. In such cases Cesium will not
+      // trigger a LEFT_UP event unless we explicitly pass a modifier.
+      (modifier) =>
+        eventHandler.setInputAction(
+          handleRelease,
+          ScreenSpaceEventType.LEFT_UP,
+          modifier as any
+        )
```

Comment:
Bind the release event + all available keyboard modifiers so that the release callback is  triggered even if the user is holding down a key.